src/ontology/run.sh.conf

```diff
@@ -1,3 +1,3 @@
 ODK_IMAGE=odkfull
-ODK_TAG=dev
+ODK_TAG=v1.5.1
```

Comment:
Good idea

Reply:
@matentzn I think it would good to pin this now and we can change again when 1.5.2 is released.